src/org/zaproxy/zap/extension/saml/SAMLUtils.java

```diff
@@ -110,26 +112,25 @@ public static String inflateMessage(byte[] data) throws SAMLException {
      * Check whether the httpMessage has a saml message in its parameters
      *
      * @param message The HttpMessage to be checked for
-     * @return whether the message has got a saml message within it
+     * @return the inspection result of the message
      */
-    public static boolean hasSAMLMessage(HttpMessage message) {
+    public static SAMLInspectionResult inspectMessage(HttpMessage message) {
```

Comment:
Worth keeping the "old" method, no need to change the codebase to call this one when the evidence is not needed.